src/scss/global.scss

```diff
@@ -180,6 +180,24 @@ button.btn {
     margin-left: auto;
     margin-right: auto;
   }
+  &.btn-primary {
+    background-image: none;
```

Comment:
What do we need the `background-image: none` for here?

Reply:
Bootstrap's `Button` component has this annoying gradient background image by default. I believe this is set somewhere in the `global.scss`, if it's not then there should be an option to disable it somehow. Anyways, I'm setting it explicitly to `none` otherwise the background color is not visible because of the image z-index.

Reply:
That makes sense. Can you please add a comment explaining that reasoning.

Reply:
Yes, of course.

Reply:
A note has been added in: https://github.com/sparkletown/sparkle/pull/1377/commits/4b7532e6f02068126c212a6f0dab1714aaaad55a